lib/core/src/server/manager/babel-loader-manager.js

```diff
@@ -16,18 +17,13 @@ export default () => ({
           require.resolve('@babel/preset-react'),
         ],
         plugins: [
-          [require.resolve('@babel/plugin-proposal-class-properties'), { loose: true }],
-          require.resolve('@babel/plugin-proposal-export-default-from'),
-          require.resolve('@babel/plugin-syntax-dynamic-import'),
-          [
-            require.resolve('@babel/plugin-proposal-object-rest-spread'),
-            { loose: true, useBuiltIns: true },
-          ],
-          require.resolve('babel-plugin-macros'),
+          ...plugins,
+          // Shoud only be done on manager to to ember
```

Comment:
![image](https://user-images.githubusercontent.com/599163/82023234-4081eb00-968e-11ea-90dc-98f2ebedd6ba.png)
I don't know much about ember, just found that it is fine without that transform.
It can't be in the common plugins

Reply:
ok